### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by surajkumarchoudhury**
problem 6 not solved..

### CODE REVIEW

File: assignments/functions.md [hunk 1]
```diff
@@ -68,34 +97,67 @@ Is there any difference in the behavior of these two variants? If there is what
 
 ```js
 // Your code goes here
+let x = +prompt("enter an integer");
```
Comment (nnnkit) [inline assignments/functions.md]: The function says to write a function that takes two parameters  returns a value. It never said you need to take the value from user using `prompt`. This portion is all that needed
```
function pow(x,n) {
  if (n <= 1) {
  return ("integers less then 1 is not allowed");
  }
  return x ** n ;
}
```

File: assignments/functions.md [hunk 2]
```diff
@@ -68,34 +97,67 @@ Is there any difference in the behavior of these two variants? If there is what
...
+  }
 ```
 
 8. 🎖Write a function `min` that takes two arguments and returns their minimum.
 
 ```js
 // Your code here.
-
+function min(x,y){
+  return math.min(x,y);
```
Comment (nnnkit) [inline assignments/functions.md:179]: It should be `Math.min` not `mat.min`

**Review by nnnkit — COMMENTED** [review]
Another exercise is not done i.e `function types`

**Comment by surajkumarchoudhury** [comment]
problem 6 solved

**Pull request merged**